operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -113,3 +113,7 @@ if [ -f ${SERVER_PID_FILE} ]; then
 fi
 
 trace "Exit script"  &>> ${STOP_OUT_FILE}
+
+trace "Stop Server: === Contents of the script output file ${STOP_OUT_FILE} ===" > /proc/1/fd/1
```

Comment:
What is the purpose of ` >> /proc/1/fd/1`?

I recommend at least adding a comment.

Reply:
K8S does not print any logs in PreStop stage. PreStop is part of lifecycle, also called as hook and generally hook logs will not print in kubectl logs. See issue [25766 ](https://github.com/kubernetes/kubernetes/issues/25766) for more details. The workaround is to print hook's output to the main process' stdout using `/proc/1/fd/1` as mentioned in the [last comment](https://github.com/kubernetes/kubernetes/issues/25766#issuecomment-752121781) in the above issue. I'll add the comment.

Reply:
I see the script is always appending to ${STOP_OUT_FILE},  what if this file is on PV?  Will it be confusing?

Reply:
ok. I'm not sure why it's always appending to ${STOP_OUT_FILE}. Do you think we should instead overwrite the ${STOP_OUT_FILE} file for each run?

Reply:
as you mentioned in the scrum, rotating it seems like a good idea - you can use the same utils helper function that we use for other log files...

Reply:
Changed in [aa5c19d](https://github.com/oracle/weblogic-kubernetes-operator/pull/3283/commits/aa5c19d141a225a9a69d5ac56af1e691c398b548).